rest/tests/base.py

```diff
@@ -772,11 +772,58 @@ def create_learning_resource_export(self, repo_slug, input_dict,
             ), resp['Location'])
             return result_dict
 
+    def get_tasks(self, expected_status=HTTP_200_OK):
+        """
+        Helper method to retrieve tasks for the user.
+        """
+        url = '/api/v1/tasks/'
+        self.assert_options_head(url, expected_status)
+        resp = self.client.get(url)
+        self.assertEqual(expected_status, resp.status_code)
+        if expected_status == HTTP_200_OK:
+            return as_json(resp)
+
+    def get_task(self, task_id, expected_status=HTTP_200_OK):
+        """
+        Helper method to retrieve task for the user.
+        """
+        resp = self.client.get(
+            '/api/v1/tasks/{task_id}/'.format(
+                task_id=task_id,
+            )
+        )
+        self.assertEqual(expected_status, resp.status_code)
+        if expected_status == HTTP_200_OK:
+            return as_json(resp)
+
+    def create_task(self, input_dict, expected_status=HTTP_201_CREATED):
+        """
+        Helper method to create a task for the user to export a tarball of
+        LearningResources.
+        """
+        resp = self.client.post(
+            '/api/v1/tasks/',
+            json.dumps(input_dict),
+            content_type='application/json',
+        )
+        self.assertEqual(expected_status, resp.status_code)
+        if expected_status == HTTP_201_CREATED:
+            return as_json(resp)
+
+    def delete_task(self, task_id, expected_status=HTTP_204_NO_CONTENT):
+        """
+        Delete a task and remove it from task list.
+        """
+        resp = self.client.delete('/api/v1/tasks/{task_id}/'.format(
+            task_id=task_id
+        ))
+        self.assertEqual(expected_status, resp.status_code)
```

Comment:
Should this test also assert that the new list of tasks is actually the one we expect, ie that it doesn't contain the one which we intended to delete, but that it does contain the rest? Overkill?